source/_integrations/russound_rio.markdown

```
@@ -68,6 +68,45 @@ The integration provides a few entities to configure the device settings. The fo
 - Loudness
 - Turn on volume
 
+## Playing media
+
+Russound RIO supports recalling AM/FM and Sirius XM presets using the `media_player.play_media` action. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)